<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QWidget" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(35, 37, 38);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(248, 211, 197);
background: transparent;
border-radius: 16px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
background: transparent;
border: none;
color: rgb(193, 189, 194);</string>
    </property>
    <property name="text">
     <string>Player 1's Turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 40px;
font-weight: bold;
color: rgb(239, 183, 170);
background: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QFrame" name="board">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="boardTiles">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="board_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="scoreTitle">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 32px;
font-weight: bold;
color: rgb(239, 183, 170);
background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="playerOneName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
color: rgba(239, 183, 170, 0.5);
background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string>Player 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="playerTwoName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
color: rgba(239, 183, 170, 0.5);
background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string>Player 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="scoreTitle_2">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>60</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
font-weight: bold;
color: rgb(239, 183, 170);
background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string>0 - 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 16px;
	font-weight: bold;
	border-radius: 8px;
}

QPushButton::hover {
	background: rgb(248, 211, 197);
	color: #252525;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="copyrightLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(236, 237, 243, 0.2);
border: none;
background: rgb(35, 37, 38);</string>
   </property>
   <property name="text">
    <string>Manuel Ferreras</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
